src/__tests__/FamilyIntake.test.js

```diff
@@ -0,0 +1,37 @@
+import FamilyIntake from '../components/common/FamilyIntake';
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+// mocking matchmedia to bypass error.
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation(query => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // deprecated
```

Comment:
If they're deprecated are we using them/needing to use them anymore? Could they be deleted?

Reply:
They are deprecated but needed for the matchMedia mock. as provided in [this post](https://jestjs.io/docs/26.x/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom).